apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -668,6 +684,23 @@ type RouteSpec struct {
 	Termination TLSRouteTerminationType `json:"termination,omitempty"`
 }
 
+// IngressSecuritySpec defines ingress security options.
+type IngressSecuritySpec struct {
+	// Type defines the security type applied to the ingress
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Type"
+	Type IngressSecurityType `json:"type,omitempty"`
+
+	// SAR defines the SAR to be used in the oauth-proxy
```

Comment:
What is the default value?

Reply:
Is empty, if you don't specify it doesn't add it to the arguments, may be we should have a default one.

Reply:
I added a default to `"{"namespace": "<tempo_stack_namespace>", "resource": "pods", "verb": "get"}`

Reply:
We should document what the default is